src/clj/game/cards/ice.clj

```diff
@@ -2310,10 +2310,9 @@
                  {:label "Runner loses 3 [credit], if able. End the run."
                   :msg "make the Runner lose 3 [credit] and end the run"
                   :async true
-                  :effect (req (if (>= (:credit runner) 3)
-                                 (wait-for (lose-credits state (make-eid state eid) :runner 3)
```

Comment:
The issue is here: `lose-credits` needs the `side` arg, which in this case will be `:runner`. Changing the impl to use `when` will break other things as it needs to be awaited.